packages/astro/src/content/content-layer.ts

```diff
@@ -137,6 +144,18 @@ class ContentLayer {
 		};
 	}
 
+	async #processMarkdown(
+		content: string,
+		options?: MarkdownProcessorRenderOptions,
```

Comment:
I'm not sure we should expose the [options](https://github.com/withastro/astro/blob/f5f33f3db51c5c8e66b0edb267353326d7f59e00/packages/markdown/remark/src/types.ts#L82)? `fileURL` is internal and i'm not sure `frontmatter` is relevant here. Maybe it makes more sense to omit it and add it if people request it